config/initializers/apartment.rb

```diff
@@ -126,8 +140,10 @@
     Application::ADMIN
   elsif request.path == "/readiness"
     Application::ADMIN
-  elsif request.path.start_with?("/lti_dynamic_registrations", "/jwks.json", "/lti_config.json", "/lti_launches/init")
-    Application::AUTH
+  # elsif request.path.start_with?("/lti_dynamic_registrations", "/jwks.json", "/lti_config.json", "/lti_launches/init")
+  #   Application::AUTH
+  elsif allowed_endpoint?(request)
```

Comment:
It's not a huge deal, but rather than having an empty clause, you could switch this to 
```
elsif !allowed_endpoint?(request)
   raise "...."
end
```

Reply:
Good idea. I made that change.